__docs__/wonder-blocks-tokens/tokens-media-queries.mdx

```diff
@@ -0,0 +1,76 @@
+import {Meta, Source} from "@storybook/blocks";
+
+import TokenTable from "../../.storybook/components/token-table";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import * as tokens from "@khanacademy/wonder-blocks-tokens";
+
+import ComponentInfo from "../../.storybook/components/component-info";
+import packageConfig from "../../packages/wonder-blocks-tokens/package.json";
+
+<Meta title="Packages / Tokens / Media Query Breakpoints" />
+
+# Media Queries
+
+<ComponentInfo name={packageConfig.name} version={packageConfig.version} />
+
+All the available media query breakpoint values and pure widths that can be used for min-width,
+max-width, width, etc.
+
+## Usage
+
+You can use mediaQueries and pure widths directly by importing `breakpoint` from the
+`wonder-blocks-tokens` package and accessing the named properties like so:
+`breakpoint.mediaQuery.sm`.
+
+### Media Queries
+
+```js
+import {breakpoint} from "@khanacademy/wonder-blocks-tokens";
+const styles = {
+    [breakpoint.mediaQuery.sm]: {
+        flexDirection: "column"
+    }
+};
+```
+
+### Pure widths
+
```

Comment:
**suggestion:** It would be nice adding a short paragraph explaining why this could be useful.

Reply:
Done!